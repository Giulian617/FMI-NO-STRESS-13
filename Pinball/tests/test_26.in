100 36157629030686
16 8 1
100 8 7
62 100 6
41 62 10
76 62 9
6 16 10
92 41 7
42 100 6
49 8 2
97 42 5
44 6 10
77 16 1
70 62 6
93 8 1
15 77 2
22 62 10
5 93 6
30 44 7
53 77 9
33 15 8
36 6 5
63 49 10
61 44 5
78 22 2
1 15 7
38 33 4
98 70 7
75 76 8
40 15 3
71 38 2
7 63 2
13 22 10
99 53 5
24 76 2
3 15 5
55 100 5
88 36 8
86 78 5
54 42 2
20 33 1
96 16 8
66 54 2
87 3 1
39 3 10
83 93 6
89 39 5
35 98 3
84 87 5
73 75 10
48 93 4
27 76 6
64 98 9
18 6 10
43 33 4
94 96 4
17 94 8
81 84 10
25 48 2
82 98 2
74 92 6
28 63 5
45 94 2
47 6 8
19 36 2
67 87 5
23 97 10
21 22 1
31 48 7
85 55 5
72 54 9
56 15 8
91 94 3
29 88 3
52 54 6
32 35 3
9 81 2
34 48 1
65 82 3
68 100 3
69 34 3
58 17 5
59 53 7
50 93 2
90 31 2
11 65 5
14 15 8
80 48 3
2 6 5
60 75 3
79 30 6
46 71 3
95 100 5
37 49 7
12 84 8
10 71 9
26 50 5
51 25 7
4 68 7
57 75 7
