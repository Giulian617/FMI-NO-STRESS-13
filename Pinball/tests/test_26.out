511362254
581455903
676608958
212613953
281746820
438906946
62023453
430562787
856157634
873330698
238109291
943953178
342458902
248252029
30478588
408433501
444630078
287562774
533339452
485093837
913568307
553936415
486538269
75662132
100849316
755757964
579767719
920481222
535411281
131653330
238109291
147057666
168436008
999588799
22672106
750852887
534562615
360686245
469174872
438259888
78389810
526824060
816127033
701427821
575628216
212613953
296622347
101778863
109294696
953293385
362702297
296622347
898567973
